matcher.py

```diff
@@ -32,7 +37,14 @@ def lexicalMatch(ref, ex, ignoreStopwords, ignoreCase):
         sRef = ref.bow().split(' ')
         sEx = ex.bow().split(' ')
         count = 0
-        
+        gold_predicate_words = [lemmatizer.lemmatize(w) for w in ref.elementToStr(ref.pred.strip()).split(' ')]
```

Comment:
Lemmatizing these predicate words as sometimes the gold extraction predicate words are conjugated (probably by the humans that created the gold solutions) and it makes the comparison later more accurate.